### PR TITLE
added recipe for prank multiple sequence aligner

### PR DESCRIPTION
**Pull request by odoublewen**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/prank/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: prank
+  version: "v.150803"
+build:
+  number: 0
+  skip: False
+source:
+  fn: prank.source.150803.tgz
+  url: http://wasabiapp.org/download/prank/prank.source.150803.tgz
+requirements:
+  build:
```
Comment (bgruening) [inline recipes/prank/meta.yaml]: Can you please add gcc/llvm as dependencies.

Comment (odoublewen) [inline recipes/prank/meta.yaml]: OK, done, thanks.

File: recipes/prank/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        - gcc # [not osx]
+        - llvm # [osx]
+  run:
+        - libgcc # [not osx]
```
Comment (bgruening) [inline recipes/prank/meta.yaml]: Is libgcc really needed?

Comment (odoublewen) [inline recipes/prank/meta.yaml]: Oh, probably not.  Removed.

**Comment by odoublewen** [comment]
Actually, I think libgcc is a runtime requirement.  The Travis CI failed without it.  I added it back in and it should pass now.

I read in [this ContinuumIO thread](https://github.com/ContinuumIO/anaconda-issues/issues/483):  `If the recipe requires gcc to build, then you must list libgcc as run dependency.`

File: recipes/prank/meta.yaml (hunk 2 above)
Comment (odoublewen) [inline recipes/prank/meta.yaml]: This recipe should be ready to package now.  Thanks!

**Comment by bgruening** [comment]
@odoublewen thanks a lot!
Let us know if you want to join this community here and see https://github.com/bioconda/bioconda-recipes/issues/1

**Comment by odoublewen** [comment]
@bgruening Thank you -- yes I would be happy to help contribute.  I have a couple more recipes I see I can help with.  I will add my name to the thread you referenced.  Thanks again.

**Comment by bgruening** [comment]
@odoublewen thanks a lot! I have added you!

**Pull request merged**